src/markdown/getting-started/install.md

```diff
@@ -6,42 +6,73 @@ position: 1
 
 First of all, you have to install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/) (standalone version is prefered) on your computer.
 
-## Install Lenra CLI using **cargo**
+Use the next commands to check if you have the right version installed:
+ni
+```bash
+# Any git version is fine
+git --version
+# A mimimum of 2.0.0 is required for docker-compose
+docker-compose --version
+```
 
-Simply install the latest package using the cargo-cli.
+You also could have the Docker Compose plugin but there is a [bug when displaying logs](https://github.com/docker/compose/issues/10179), so we recommend you to use the standalone version.
```

Comment:
When we look at the issue you linked here, it seems this is not a but from docker compose but from Lenra that didn't send the SIGTERM to the right PID.

Reply:
Actually we don't have acces to the child PID of the docker process... So it's a Docker plugin issue.